distance.go

```diff
@@ -7,77 +7,46 @@ package units
  */
 
 import (
+	"fmt"
 	"math"
 	"strconv"
 	"strings"
 )
 
-// Distance represents a quantity of length.
-// Internally, it stores the distance in metres, as a float64.
-// Create a new Distance by multiplying one of the constants:
-// d1 := 100 * units.Metre
-// d2 := 10 * units.NauticalMile
-// Use methods like Metres() or Kilometres() to get the distance in the unit of your choice.
-// US and UK spelling of consts and methods supported.
-
-type Distance float64
+// Distance represents a quantity of speed.
+type Distance interface {
+	Unit
```

Comment:
I assume this is a comment?

Reply:
No, it adds the `Unit` interface, defined in `unit.go`, which I forgot to `git add`. It's there now